common/main/af.xml

```diff
@@ -1024,11 +1024,11 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 		</codePatterns>
 	</localeDisplayNames>
 	<characters>
-		<exemplarCharacters>[a á â b c d e é è ê ë f g h i î ï j k l m n o ô ö p q r s t u û v w x y z]</exemplarCharacters>
-		<exemplarCharacters type="auxiliary">[à å ä ã æ ç í ì ó ò ú ù ü ý]</exemplarCharacters>
+		<exemplarCharacters>[aáâ b c d eéèêë f g h iîï j k l m n oôö p q r s t uû v w x y z]</exemplarCharacters>
+		<exemplarCharacters type="auxiliary">[àåäã æ ç íì óò úùü ý]</exemplarCharacters>
```

Comment:
I don't know the rationale for spaces in and around "b c d" but no spaces in "aáâ"

Does this rule only apply in the xml but not in the ST interface? It doesn't seem to be mentioned at https://cldr.unicode.org/translation/core-data/exemplars

Reply:
@macchiati is this related to the exemplar set display changes?